hazelcast/src/main/resources/hazelcast-config-3.10.xsd

```diff
@@ -751,6 +751,7 @@
                     </xs:sequence>
                 </xs:complexType>
             </xs:element>
+            <xs:element name="quorum-ref" minOccurs="0" maxOccurs="1"/>
```

Comment:
Can you add some documentation for all of these?

Reply:
Done